### DISCUSSION TITLE
Prior

### DISCUSSION THREAD

**Discussion by msiemionCalistapw** [moj-analytical-services/splink#854] [Q&A]
How can I find an accurate prior? 

I have been basically plugging in random fractions for "probability_two_random_records_match" and it seems like with even slight differences the prior changes drastically, making it hard to find a prior that accurately affects match scores.

**Comment by RobinL** [comment]
https://moj-analytical-services.github.io/splink/linker.html#splink.linker.Linker.estimate_probability_two_random_records_match usually will give you best results

**Reply by msiemionCalistapw** [reply]
I did try this but got the error attached below
![Uploading Capture.PNG…]()
